app/policies/reifications_policy.rb

```diff
@@ -0,0 +1,11 @@
+class ReificationsPolicy < ApplicationPolicy
+  def create?
+    own_record?
+  end
+
+  private
+
+  def own_record?
+    @record.reify.user == @user
```

Comment:
(We can't do `@record.reify.user_id == @user.id` here because the user might be a "through" association, i.e. there might not be a `user_id` column on the model in question.)